uavcan/node/port/SubjectIDList.1.0.dsdl

```diff
@@ -0,0 +1,19 @@
+# A list of subject identifiers.
+# The range of subject-ID is large, so using a fixed-size bitmask would make this type difficult to handle on
+# resource-constrained systems. To address that, we provide two extra options: a simple variable-length list,
+# and a special case that indicates that every subject-ID is in use.
+
+@union
+
+uint16 CAPACITY = SubjectID.1.0.MAX + 1
+
+bool[CAPACITY] mask
```

Comment:
If I use the sparse_list am I required to also set the mask? There's a bit of ambiguity here on the requirements for using this type.